src/app/tracing/dialog/station-properties/station-properties.component.ts

```diff
@@ -211,7 +215,7 @@ export class StationPropertiesComponent implements OnInit, OnDestroy {
     private initProperties(station: StationData, columns: TableColumn[]): void {
         const properties: Properties = {};
         const hiddenProps = Utils.createSimpleStringSet(this.notListedProps);
```

Comment:
Please add row: 
`this.conditionalNotListedPropsMap.forEach((isHiddenFun, propId) => hiddenProps[propId] = isHiddenFun(station));`
Please remove:
`&& !this.specialProps.has(key)`

Reply:
done!